src/screens/CreatePost/index.js

```diff
@@ -78,8 +79,6 @@ import {requestCameraPermission, requestExternalStoragePermission} from '../../u
 import {uploadPhoto} from '../../service/file';
 import ImageUtils from '../../utils/image';
 import ImageCompressionUtils from '../../utils/image/compress';
-import {COLORS} from '../../utils/theme';
-import {useDynamicColors} from '../../hooks/useToggleColors';
 
 const IS_GEO_SELECT_ENABLED = false;
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `COLORS` import and the `useDynamicColors` hook might affect the dynamic color functionality of the application. Make sure this change doesn't break any existing functionality.
```diff
- import {COLORS} from '../../utils/theme';
- import {useDynamicColors} from '../../hooks/useToggleColors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->